src/main/java/com/github/jikoo/regionerator/Regionerator.java

```diff
@@ -237,6 +237,11 @@ void finishCycle(@NotNull DeletionRunnable runnable) {
 	 * Attempts to activate {@link DeletionRunnable}s for any configured worlds.
 	 */
 	public void attemptDeletionActivation() {
+		
+		if (!(getConfig().getBoolean("deletion.enable"))) {
```

Comment:
Config options should not be used raw like this - they should be in the Config class with an accessor method for easy management. Also unnecessary here as we'd be piggybacking the existing pause state, should just set that from config once during enable.